WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -21,6 +21,48 @@
     public static final String GB_EDITOR_NAME = "gutenberg";
     public static final String AZTEC_EDITOR_NAME = "aztec";
 
+    /**
+     * Migrate the old app-wide editor preference value to per-site setting. DotCom sites will make a network call
+     * and store the value on the backend. DotORG sites just store the value in the local DB in FluxC
+     *
+     * Strategy: Check if there is the old app-wide preference still available (v12.9 and before used it).
+     * -- 12.9 ON -> turn all sites ON in 13.0
+     * -- 12.9 OPTED OUT (were auto-opted in but turned it OFF) -> turn all sites OFF in 13.0
+     *
+     * @param dispatcher FluxC dispatcher
+     * @param siteStore  SiteStore
+     */
+    public static void migrateAppWideMobileEditorPreferenceToRemote(final Dispatcher dispatcher,
+                                                                    final SiteStore siteStore) {
+        if (!AppPrefs.isDefaultAppWideEditorPreferenceSet()) {
+            return;
```

Comment:
As per the "OPTED OUT" case mentioned in the function comment, we need to check if the user *wasn't* offered auto-enable before to just return, right?

Reply:
Right, if the user didn't open a block based post in a previous version of the app, the preference is not there in the app. We will show the opt-it popup as soon as the user open a block based post.